Add an article on how to Stop validating (#2000)
w3f/polkadot-wiki: docs/maintain-guides-how-to-stop-validating.md

---
id: maintain-guides-how-to-stop-validating
title: How to Stop Validating
sidebar_label: How to Stop Validating
---

To ensure a smooth stop to validation, make sure you should do the following actions:

> 1. Chill your validator tokens

Reviewer: ```suggestion
1. Chill your validator
```

The chilling is done on the validator, the tokens are still bonded in this case. In most cases I would say `chill`-ing refers to being a validator or nominator, not the tokens.

2. Purge validator session keys
3. Unbond your tokens

These can all be done with [PolkadotJS Apps](https://polkadot.js.org/apps) interface or with
extrinsics.

> ## Chill tokens

Reviewer: ```suggestion
## Chill Validator
```

> To chill your tokens, see the [How to Chill](maintain-guides-how-to-chill) page. You can also

Reviewer: ```suggestion
To chill your validator or nominator, see the [How to Chill](maintain-guides-how-to-chill) page. You can also
```

[claim your rewards](learn-simple-payouts#claiming-rewards) at this time.

> 

Reviewer: Would add another step here, something like:

"If you wish to remain a validator or nominator, submitting the `chill` extrinsic should suffice. It is only if you wish to unbond funds or reap an account that you should continue with the following."

I mention this because a common use case for stopping a validator is planned downtime and server maintenance, where a validator might stop their active role to change things on their server, instead of rolling over to a backup node. And in this case, they would not unbond or purge keys.

Author: Awesome, I've added that quote between "Chill your validator" and "Purge validator session keys" in the outline at the top of the page; that seems like a better spot for it.
